Add a section about collecting logs through HTTP (#211)
CleverCloud/documentation: content/doc/administrate/log-management.md

For more information, please refer to the [official documentation](https://www.elastic.co/guide/en/elasticsearch/reference/current/set-up-lifecycle-policy.html).

### HTTPS

> There is a side project, available on GitHub which enables to create a drain to collect logs through an HTTPS endpoint. It is fully compatible with Clever Cloud.

Reviewer: ```suggestion
Some tools available on GitHub enable to create a drain to collect logs through an HTTPS endpoint. [This project](https://github.com/sebartyr/http-logs-drain), for example, is fully compatible with Clever Cloud 
```

You could host it as an app and an add-on on Clever Cloud. A complete README explains all the features.

> You can find the repository of the project [here](https://github.com/sebartyr/http-logs-drain).

Reviewer: ```suggestion
```
